### PR TITLE
tranlated main and game

### LINKED ISSUE #10: Make Bengali locale
Use the English (`en.json`) translations from the i18n folder in assets with the exception of the shell folder. Each file name should be named `bn.json`
https://github.com/tedaky/memory-game/tree/master/projects/the-application/src/assets/i18n

### PR DESCRIPTION
**Pull request by no9600hp**
translated game and main to bengali

Closes #10

### CODE REVIEW

File: projects/the-application/src/assets/i18n/main/bn.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+  "FULL_STOP": "ফুলস্টপ্",
+  "GAME": "খেলা",
+  "HIGH_SCORES": "সেরা স্কোর",
+  "INSTALL": "ইনস্টল",
+  "INSTALL_MEMORY_GAME": "মেমরি গেম ইনস্টল করুন",
+  "PLEASE_RELOAD": "লোড করুন",
+  "PLEASE_FINISH_OR_RESET_GAME": "খেলা শেষ বা পুনরায় সেট করুন",
+  "RECENT_SCORES": "সাম্প্রতিক স্কোর",
+  "REFRESH": "রিফ্রেশ",
+  "SETTINGS": "সেটিংস",
```
Comment (tedaky) [inline projects/the-application/src/assets/i18n/main/bn.json:14]: Can you add TAP and THEN?

See https://github.com/tedaky/memory-game/pull/16/files#diff-47c31de70082a49e837b88ef47ba61dc365e3d92008c6d09cbacc1a0edbfb98a

**Review by tedaky — CHANGES_REQUESTED** [review]
Add TAP and THEN to main/bn.json

File: projects/the-application/src/assets/i18n/main/bn.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-  "PLEASE_RELOAD": "PLEASE_RELOAD",
-  "PLEASE_FINISH_OR_RESET_GAME": "PLEASE_FINISH_OR_RESET_GAME",
-  "RECENT_SCORES": "RECENT_SCORES",
-  "REFRESH": "REFRESH",
-  "SETTINGS": "SETTINGS",
-  "TITLE": "TITLE",
-  "UPDATE_AVAILABLE": "UPDATE_AVAILABLE"
+  "ADD_TO_HOME_SCREEN": "হোম পর্দায় যোগ করুন",
+  "CLOSE": "বন্ধ",
+  "DESCRIPTION": "বর্ণনা",
```
Comment (tedaky) [inline projects/the-application/src/assets/i18n/main/bn.json:4]: Can you translate the description from the English `DESCRIPTION` in `en.json`?

File: projects/the-application/src/assets/i18n/main/bn.json [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+  "GAME": "খেলা",
+  "HIGH_SCORES": "সেরা স্কোর",
+  "INSTALL": "ইনস্টল",
+  "INSTALL_MEMORY_GAME": "মেমরি গেম ইনস্টল করুন",
+  "PLEASE_RELOAD": "লোড করুন",
+  "PLEASE_FINISH_OR_RESET_GAME": "খেলা শেষ বা পুনরায় সেট করুন",
+  "RECENT_SCORES": "সাম্প্রতিক স্কোর",
+  "REFRESH": "রিফ্রেশ",
+  "SETTINGS": "সেটিংস",
+  "TITLE": "শিরোনাম",
```
Comment (tedaky) [inline projects/the-application/src/assets/i18n/main/bn.json:15]: Can you translate the title from the English `TITLE` in `en.json`?

File: projects/the-application/src/assets/i18n/main/bn.json [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-  "PLEASE_FINISH_OR_RESET_GAME": "PLEASE_FINISH_OR_RESET_GAME",
-  "RECENT_SCORES": "RECENT_SCORES",
-  "REFRESH": "REFRESH",
-  "SETTINGS": "SETTINGS",
-  "TITLE": "TITLE",
-  "UPDATE_AVAILABLE": "UPDATE_AVAILABLE"
+  "ADD_TO_HOME_SCREEN": "হোম পর্দায় যোগ করুন",
+  "CLOSE": "বন্ধ",
+  "DESCRIPTION": "বর্ণনা",
+  "FULL_STOP": "ফুলস্টপ্",
```
Comment (tedaky) [inline projects/the-application/src/assets/i18n/main/bn.json:5]: From what @haldarsumana told me there is a different symbol for `.` (period) in Bengali. Can you set the value of `FULL_STOP` to that symbol?

**Review by tedaky — CHANGES_REQUESTED** [review]
Please provide the translation for `TITLE`, `DESCRIPTION` and `FULL_STOP`

File: projects/the-application/src/assets/i18n/main/bn.json (hunk 1 above)
Comment (no9600hp) [reply projects/the-application/src/assets/i18n/main/bn.json:14]: added

**Review by haldarsumana — APPROVED** [review]

**Review by tedaky — APPROVED** [review]

**Pull request merged**